src/webserver/Listener.h

```diff
@@ -239,12 +239,20 @@ class Listener
         // Bind to the server address
         acceptor_.bind(endpoint, ec);
         if (ec)
-            return;
+        {
+            BOOST_LOG_TRIVIAL(error)
+                << "Failed to bind to endpoint: " << endpoint;
+            throw std::runtime_error("Failed to bind to specified endpoint");
```

Comment:
Do you think it may be worth including `ec.message()` somewhere? 
I'm approving anyway and will let you decide if you want it or not.